pkg/monitoring/alloy/templates/monitoring-config.yaml.template

```diff
@@ -0,0 +1,83 @@
+# This file was generated by observability-operator.
+# It configures Alloy to be used as a monitoring agent.
+# - configMap is generated from logging.alloy.template and passed as a string
+#   here and will be created by Alloy's chart.
+# - Alloy runs as a statefulset, with required tolerations in order to scrape metrics
+networkPolicy:
```

Comment:
Do we need the netpol here? Would it not be best to configure it in the bundle?

Reply:
I would rather keep all the values in the same place and not split it in order to make thing simpler to understand.

Reply:
I agree about the config itself but not really about netpols and stuff though :( I do not think it make sense as those concerns should not be addressed by the observabliyt operator but the app itself

Reply:
I think the configuration surface should be kept to a minimum but we can change that later

Reply:
Yes I understand having this closer to the app makes sense. As you said we can change this later on.